beesdoo_product/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+{
+    'name': "beesdoo_product",
+
+    'summary': """
+        Modification of product module for the needs of beescoop
+        - SOOO5 - Ajout de label bio/ethique/provenance""",
+
+    'description': """
+
+    """,
+
+    'author': "Beescoop - Cellule IT",
+    'website': "https://github.com/beescoop/Obeesdoo",
+
+    # Categories can be used to filter modules in modules listing
+    # Check https://github.com/odoo/odoo/blob/master/openerp/addons/base/module/module_data.xml
+    # for the full list
+    'category': 'Sales Management',
+    'version': '0.1',
```

Comment:
change to 12.0.1.0.0

Reply:
maybe add coop it easy in the contributor as part of the works has been made by CIE workers non volunteering to bees coop